tf2_msgs/package.xml

```diff
@@ -10,17 +11,18 @@
   <license>BSD</license>
 
   <url type="website">http://www.ros.org/wiki/tf2_msgs</url>
-    
-  <buildtool_depend>catkin</buildtool_depend>
 
-  <build_depend>actionlib_msgs</build_depend>
-  <build_depend>geometry_msgs</build_depend>
-  <build_depend>message_generation</build_depend>
+  <buildtool_depend>ament_cmake</buildtool_depend>
+  <buildtool_depend>rosidl_default_generators</buildtool_depend>
 
-  <run_depend>actionlib_msgs</run_depend>
-  <run_depend>geometry_msgs</run_depend>
-  <run_depend>message_generation</run_depend>
+  <!-- <depend>actionlib_msgs</depend> -->
+  <depend>builtin_interfaces</depend>
+  <depend>geometry_msgs</depend>
 
-</package>
+  <exec_depend>rosidl_default_runtime</exec_depend>
```

Comment:
Conveniently we just added this package in https://github.com/ros2/rosidl/pull/57